ambuda/__init__.py

```diff
@@ -86,9 +70,26 @@ def create_app(config_env: str):
     # Config
     app.config.from_object(config_spec)
 
+    # Setup logger config before initializing other endpoints
+    Logger.set_log_config(config_spec)
+
     # Database
     _initialize_db_session(app, config_env)
 
+    # Lazy import modules to add as endpoints
+    from ambuda import auth as auth_manager
```

Comment:
Lazy imports seems to add more complexity, but I'm not sure we need it. After all, we only call `set_log_config` once.

What if we instead:
- Use regular imports, not lazy imports here, and
- Removed `set_log_config`, and just set the default log level in the initialization of `Logger` in `logger.py`?